packages/holocron-dev-server/src/utils/statics.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import path from 'path';
+import { execSync, spawnSync } from 'child_process';
+
+import { ufs } from './virtual-file-system';
+import { getOneAppPath, getStaticPath, getTempPath } from './paths';
+import {
+  error,
+  logGitIgnoreAddition,
+  logPullingDockerImage,
+  logOneAppVersion,
+  logStaticStep,
+} from './logs';
+import { libraryName, oneAppDockerImageName } from '../constants';
+
+export function addStaticsDirToGitIgnore() {
+  // adds static/ to .gitignore only once
+  const gitIgnorePath = path.resolve('.gitignore');
+  if (ufs.existsSync(gitIgnorePath)) {
+    const gitIgnoreAddition = [`# added by ${libraryName}`, 'static/'].join('\n');
+    if (!ufs.readFileSync(gitIgnorePath).toString().includes(gitIgnoreAddition)) {
+      logGitIgnoreAddition();
+      execSync(`echo "\n${gitIgnoreAddition}" >> ${gitIgnorePath}`);
+    }
+  }
+}
+
+export function loadOneAppStaticsFromDocker({
+  tempDir = getTempPath(),
+  appDir = getOneAppPath(),
+  dockerImage = oneAppDockerImageName,
```

Comment:
Yes by default the One App statics are using the latest `oneamex/one-app-dev:latest` image, however if a module does provide `one-amex.runner.dockerImage` with a specific version, that will be used instead of the latest version.

Reply:
I will update to document this.